src/Uno.UI/UI/Xaml/Controls/ProgressRing/ProgressRing.iOS.cs

```diff
@@ -40,7 +40,7 @@ protected override void OnIsEnabledChanged(bool oldValue, bool newValue)
 
 		partial void OnIsActiveChangedPartial(bool _) => TrySetNativeAnimating();
 
-		private void TrySetNativeAnimating()
+		partial void TrySetNativeAnimating()
 		{
 			if (_native != null)
 			{
```

Comment:
Shouldn't we remove the check to IsEnabled ? In  `if (IsActive && IsEnabled && IsLoaded)`

Reply:
Yes, it was supposed to be a CI validation, but the commits were invalid :)